website/source/docs/vault-enterprise/replication/index.html.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: "docs"
+page_title: "Vault Enterprise HSM Support"
+sidebar_current: "docs-vault-enterprise"
+description: |-
+  Vault Enterprise has support for Replication, allowing critical data to be replicated across clusters. 
+
+---
+
+# Vault Replication
+
+## Overview
+
+Many organizations have infrastructure that spans multiple datacenters. Vault provides 
+the critical services of identity management, secrets storage, and policy management. 
+This functionality is expected to be highly available and operators would like to ensure
+a common set of policies are enforced globally, and a consistent set of secrets and keys
+are exposed to applications that need to interoperate. Vault replication enables
+redundancy across datacenters.
+
+Prior to Vault 0.7, Vault nodes could only be paired within clusters using a common HA
+storage backend. A single node is locked with a set of other nodes in a cluster to ensure
+high availability. With replication, we intend to allow Vault to allow horizontal 
+scalability between clusters a cross geographically distributed data centers.
+
+## Architecture
+
+When replicating clusters, a single cluster is designed the primary cluster. The
+primary cluster is the active cluster in use within an infrastructure, which 
+serves as the system  of record for replication.
+
+The primary cluster asynchronously replicates data to a series of remote standby
+clusters. These clusters are known as secondary clusters or secondaries. 
+
+Roles for primaries and secondaries, as well as authentication for each and the process
+of replicating data between them, is setup using replication tokens which are exchanged
+during bootstrapping. Once setup, replication uses end-to-end TLS, and Vault manages the
+lifecycle of certificates.
+
+## What Is Replicated?
+
+The data replicated in Vault 0.7 will include:
+
+ * Secrets
+ * Policies
+ * Configuration details for secret backends
+ * Configuration details for authentication backends
+ * Configuration details for audit backends
+
+Access tokens for secrets are not replicated during the replication process, as tokens are 
+local to a cluster that has generated them.  
+
+## Activating Replication 
+
+### Activating the Primary
+
+To activate the primary, run vault write -f sys/replication/primary/enable. 
+
+There is currently one optional argument: primary_cluster_addr. This can be used to override
+the cluster address that the primary advertises to the secondary, in case the internal network
+address/pathing is different between members of a single cluster and primary/secondary clusters.
+
+### Fetching a Secondary Token
+
+To fetch a secondary bootstrap token, run vault write sys/replication/primary/secondary-token id=<id>. 
+
+The value for ID is opaque to Vault and can be any identifying value you want; this can be used
```

Comment:
I would recommend unique ID's (even if not required) so that you can uniquely identify which secondary to revoke access for.